etc/config/fortran.defaults.properties

```diff
@@ -0,0 +1,8 @@
+# Default settings for Fortran
+compilers=/usr/bin/gfortran-4.4:/usr/bin/gfortran-4.5:/usr/bin/gfortran-4.6:/usr/bin/gfortran-4.7:/usr/bin/gfortran-4.8:/usr/bin/gfortran-5:/usr/bin/gfortran-6:/usr/bin/gfortran-7:/usr/bin/gfortran-8:/usr/bin/gfortran-8.1:/usr/bin/gfortran:/usr/bin/flang
```

Comment:
I think we are moving past defining absolute paths for the default compilers, and instead relying on whatever can be found on the `PATH` env variable. (@jaredwy)

Note that the file `fortan.amazon.properties` is also missing. This is the file which gets picked up by the Amazon instances. The compiler paths are a bit trickier to find without a proper setup, but you can take a look at `c++.amazon.properties` to see what they look like.

Reply:
I've modeled `fortran.default.properties` after `c.defaults.properties`. Conceptually I understand why absolute paths are not nice, but all the files I checked seemed to have them? So I probably misunderstand something here. ... Ok, I guess I see it now. It's the default one that references a later definition with absolute paths?
I'll give it a shot. Tomorrow.

Reply:
How would I find out if `ifort` is installed along with `icc` and `icpc` on this Amazon thing? Is the complete compiler suite available?

Reply:
Yeah, the old defaults use paths. We didn't change those to be on the safe side of compatibility with 3rd parties.

As for the paths, it gets trickier. Let me check on my local instance. I remember that  @mattgodbolt said gcc fortan compilers are being built already. 

 (Note that if you want, running a script from the sister repo will download ~60GB of compilers. This compilers get sent to the same path as in amazon)

Reply:
It seems as if most gcc installations have a `gfortan` included. You can start by adding those (The path should be the same that for the gcc bins, but pointing to `gfortan` instead)

`ifort` does not seem to be part of the current Intel compiler installations.

Reply:
I answered a relevant question some weeks ago: https://github.com/mattgodbolt/compiler-explorer/issues/929#issuecomment-390112479 might prove useful to you too 👍

Reply:
Ok, I've tried adding the amazon file and remove the absolute path from the other. I kept GCC for ARM and Power assuming if they're not found it doesn't hurt, if they're found they would show up.